etc/conf/ssh_config

```diff
@@ -13,6 +13,9 @@ Host _remote*
 Host _remote_background_indep_tcp
     Port 42220
 
+Host _remote_background_indep_ssh
+    Port 42220
+
```

Comment:
Is this supposed to be different to `_remote_background_indep_tcp`?

Reply:
Same port - Oliver told me on Friday :)

Reply:
Yep, SSH defaults to use the TCP protocol anyway so we can use the same container for both.

> **Note:** We use a different container for the polling tests which should not be capable of TCP comms just to make sure. It's a bit over the top but should give us good confidence that polling works correctly.

I've added `_remote_background_indep_ssh` as a separate host to fit in with the same approach (easier to handle in actions, etc) but we could just use the same host here.

We could add another container (duplicate of the TCP one) to the swarm to fully separate them.